### ISSUE TITLE
Vavoo 6.0

### ISSUE THREAD

**Issue by danilp** [michaz1988/michaz1988.github.io#7]
Hi, addon vavoo 6.0 is not working for me. Same as  v.5.3.6 channels don't play....

**Comment by michaz1988** [comment]
could you post a log

**Comment by danilp** [comment]
File "/storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/script.module.requests/lib/requests/adapters.py", line 517, in send
                                                       raise SSLError(e, request=request)
                                                   requests.exceptions.SSLError: HTTPSConnectionPool(host='www.vavoo.tv', port=443): Max retries exceeded with url: /api/box/ping2 (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:992)')))
                                                   -->End of Python script error report<--
                                                   
2023-10-28 14:24:22.629 T:20038   error <general>: Playlist Player: skipping unplayable item: 0, path [plugin://plugin.video.vavooto/?name=DAZN+LALIGA]
2023-10-28 14:24:26.193 T:20038    info <general>: CWinSystemAndroid::DestroyWindow
2023-10-28 14:24:26.405 T:20001 warning <general>: CPeripheralBusAndroid: failed to update input device with ID 4 because it couldn't be found

**Comment by danilp** [comment]
Kodi android monbile and firetvstick

**Comment by michaz1988** [comment]
are you using a vpn ? SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] means your ssl certificat is not accepted

**Comment by danilp** [comment]
No, normaly i don't need It. Last week i used the plugin without problema...

**Comment by michaz1988** [comment]
then maybe your kodi version has a problem pls try reinstall

**Comment by danilp** [comment]
Ok, but I and my Friends have same problem...

**Comment by michaz1988** [comment]
> Ok, but I and my Friends have same problem...

its strange i have kodi 20.2, 19.5 and even 18 installed on android and firetv and it works

**Comment by danilp** [comment]
I've 20.2 on android and firetvstick...same result nok... Thanks a lot!

**Comment by danilp** [comment]
Hi, I have downloaded the app vavoo from the vavoo.to web. When I start the app, I receive the same SSL error... 
![Screenshot_2023-10-29-10-28-32-074_tv vavoo app](https://github.com/michaz1988/michaz1988.github.io/assets/17127599/f5c8ca55-9084-471e-8e5b-d87a9f697081)
